chart/templates/nginx.yaml

```diff
@@ -12,8 +12,8 @@ data:
     stream {
         map $ssl_preread_server_name $selected_upstream {
             hostnames;  
-            *.admin.uds.dev backend_admin;
-            *.uds.dev backend_tenant;
+            *.admin.{{ .Values.domain }} backend_admin;
```

Comment:
uds-core now/soon will support a custom admin domain (as of 0.33.0, see [here](https://uds.defenseunicorns.com/reference/uds-core/dns/#domain-configuration)). Might be good to get ahead of that in case it is asked and add a value for `adminDomain`.

Also wondering if we should set these as zarf vars with the same names (`DOMAIN` and `ADMIN_DOMAIN`) so that someone could modify domain for k3d and core all with one var?

Reply:
There is also another possibility here that gets more messy, but someone could technically set domain and admin domain both to `uds.dev`. I think we're good to just have that not be supported in uds-k3d, just wanted to call it out here for note.

Reply:
good call.  i refactored to prep for the addition of admin domain and set up zarf variables for these guys instead.  let me know how it looks

Reply:
oh wait. I am missing something.  trying to mirror the pattern in core.  how do you get the admin domain zarf variable default to be admin.DOMAIN?

Reply:
nvm. I think i see it now.  using helm templates, if the zarf var `ADMIN_DOMAIN` is not set, just default to `admin.DOMAIN`. Lmk if this approach is sane.

Reply:
I think this works, you'll need to move the zarf vars for the domain values to `values/dev-stack-values.yaml` in order for them to properly be used by the helm templating.

Reply:
i seeee.  just pushed a fix